modules/hosted-restart-hcp-components.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assembly:
+//
+// * hosted_control_planes/index.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-restart-hcp-components_{context}"]
+= Restarting hosted control plane components
+
+If you are an administrator for hosted control planes, you can use the `hypershift.openshift.io/restart-date` annotation to restart all control plane components for a particular `HostedCluster` resource. For example, you might need to restart control plane components for certificate rotation.
+
+.Procedure
+
+To restart a control plane, annotate the `HostedCluster` resource by entering the following command:
+
+[source,terminal]
+----
+$ oc annotate hostedcluster -n clusters example hypershift.openshift.io/restart-date=$(date --iso-8601=seconds)
+----
+
+//lahinson - june 2024 - are any of the values in the above command user-replaced variables? If so, we need to format them in angle brackets and provide instructions for what the user should replace them with.
```

Comment:
Did you get an answer here? 
If so, would an example date string help (it would help me!!).

Reply:
Thanks for the reminder! I got an answer and updated the command to indicate where the user needs to replace values.